go/filemanager/pfsmodules/chunk.go

```diff
@@ -3,23 +3,24 @@ package pfsmodules
 import (
 	"errors"
 	"fmt"
-	"io"
 	"net/url"
-	"os"
 	"strconv"
-
-	log "github.com/golang/glog"
 )
 
-// Chunk respresents a chunk info.
-type Chunk struct {
+// ChunkParam respresents a chunk info.
+type ChunkParam struct {
 	Path   string
 	Offset int64
 	Size   int64
 }
 
+// ToString packs info of ChunkParam.
+func (p *ChunkParam) ToString() string {
+	return fmt.Sprintf("Path:%s Offset:%d Size:%d", p.Path, p.Offset, p.Size)
+}
+
 // ToURLParam encodes variables to url encoding parameters.
-func (p *Chunk) ToURLParam() url.Values {
+func (p *ChunkParam) ToURLParam() url.Values {
```

Comment:
Not a requirement, but it seems a lot of effort to write `ToURLParam` and `ParseChunkParam` manually for each type that we want to pass over the wire. If we use json, the json encoder / decoder can do these automatically for us, and always correctly.